sessionctx/variable/sysvar.go

```diff
@@ -1791,6 +1791,10 @@ var defaultSysVars = []*SysVar{
 			}
 			return nil
 		}},
+	{Scope: ScopeSession, Name: TiDBEnableTiFlashReadForWriteStmt, Value: BoolToOnOff(DefTiDBEnableTiFlashReadForWriteStmt), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
```

Comment:
ScopeGlobal is not needed?

Reply:
ScopeSession is enough for testing at present.